DirectEmployers/MyJobs
# [PD 3181] edit distribution/rule dropdown broke

## What's this PR do?
PR fixes the distribution and rules dropdowns for them to re-populate and also submit when edited inside of a manage contacts page

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO Application

## How should this be manually tested?
Pull down the repository. Go into the CBO application and go to a CBO. When you are on the CBO Manage Contact page for a specific contact, if you go and edit the distribution location or go to edit a rule you should see the dropdowns populate with what's currently set. Also once changed, you will now be able to update without the dropdowns throwing an error.

## Any background context you want to provide?
